src/main/java/fi/helsinki/cs/tmc/core/commands/SendFeedback.java

```diff
@@ -6,6 +6,7 @@
 
 import com.google.gson.JsonArray;
 import com.google.gson.JsonObject;
+import java.net.URI;
 
 import java.util.Map;
```

Comment:
Nit: The empty line should be between the `java` and `com.google` import groups, not inside the `java` group.